go.mod

```diff
@@ -3,44 +3,38 @@ module github.com/tendermint/tendermint
 go 1.12
 
 require (
-	github.com/VividCortex/gohistogram v1.0.0 // indirect
-	github.com/beorn7/perks v0.0.0-20180321164747-3a771d992973 // indirect
-	github.com/btcsuite/btcd v0.0.0-20190115013929-ed77733ec07d
-	github.com/btcsuite/btcutil v0.0.0-20180706230648-ab6388e0c60a
-	github.com/fortytw2/leaktest v1.2.0
-	github.com/go-kit/kit v0.6.0
+	github.com/btcsuite/btcd v0.0.0-20190629003639-c26ffa870fd8
```

Comment:
We shouldn't update dependencies all at once, but rather do it one by one plus a review of the changes